### DISCUSSION TITLE
portout/portin skinparams?

### DISCUSSION THREAD

**Discussion by dzanot** [plantuml/plantuml#1095] [Q&A] [answered]
I'm having trouble styling ports, is there a special skinparam for them not documented?

This is showing a successful component styling and an unsuccessful portout styling

```plantuml
@startuml

title Interfaces - Component Diagram

[Business Logic]
[Data Access] as DA  
component "Graphic User\nInterface" as GUI {
  portout "Out" as out
}
skinparam component {
  color green
  BackgroundColor red
  BorderColor blue
}
skinparam portout {
  color green
  BackgroundColor red
  BorderColor blue
}
interface IMath as Math
interface "IItems" as Items

[Business Logic] -- Math
DA .. Items

@enduml
```

**Comment by The-Lum** [comment, marked as answer]
Hello @dzanot, and all,

>Is there a special skinparam for them not documented?

Yes: Currently `port` is assimilated as `boundary`...

Perhaps you can test, with new version of PlantUML (V1.2022.7):
```puml
<style>
boundary {
  FontColor green
  BackgroundColor red
  LineColor blue
}
</style>

```
See also here:
- [QA-16306](https://forum.plantuml.net/16306/coloring-single-ports)
- [Changes & Release note](https://plantuml.com/changes)

**Reply by dzanot** [reply]
Excellent, thank you @The-Lum !
